### PR TITLE
Provide a better experience when on the project settings tab when Bia…

### PR DESCRIPTION
**Pull request by pnaik1**
Closes: #2024 

## Description
Hide the settings tab on the project details page when bias metrics are disabled by feature flag

## How Has This Been Tested?
diable the BiasMetrics feature flag and check whether the the setting tab is hidden

Screenshot from 2023-11-01 13-59-24

## Test Impact
No test here are no tests as yet for this feature.

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [x] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by ppadti** [comment]
working fine.
/lgtm

Screenshot from 2023-11-01 19-00-02

**Comment by DaoDaoNoCode** [comment]
Might be related to #2032, not sure which one will be merged first, but the other one will need to use its work.

File: frontend/src/pages/projects/projectSharing/utils.ts
Comment (alexcreasy) [inline frontend/src/pages/projects/projectSharing/utils.ts:31]: This function isn't necessary, we already have the `useBiasMetricsEnabled` hook[1] please could you update the PR to use this?

[1] https://github.com/alexcreasy/odh-dashboard/blob/e2f799d38ee6b18c2d224722a6a6fd875a1dd6ca/frontend/src/concepts/explainability/useBiasMetricsEnabled.ts

**Review by alexcreasy — CHANGES_REQUESTED** [review]
Looks good, one change requested though.

File: frontend/src/pages/projects/screens/detail/ProjectDetails.tsx
Comment (alexcreasy) [reply frontend/src/pages/projects/screens/detail/ProjectDetails.tsx:33]: @christianvogt Thank for raising this. We're looking to merge `f/mserving-metrics` into incubation tomorrow (7th November), so as this is against that branch, I'd propose we merge this as is (after Purva makes the requested updates to use the existing hook you mentioned).

As #2032 target's main, they won't immediately conflict, I propose I create an issue to update the `f/mserving-metrics` features once #2032 is merged? 

Any objections?

Comment (christianvogt) [reply frontend/src/pages/projects/screens/detail/ProjectDetails.tsx]: No issues with your work going first. I assumed the #2032 might get in first.

**Review by alexcreasy — APPROVED** [review]

**Pull request merged**
